test/functional/feature_assumeutxo.py

```diff
@@ -395,6 +395,10 @@ def check_tx_counts(final: bool) -> None:
         assert_equal(snapshot['snapshot_blockhash'], dump_output['base_hash'])
         assert_equal(snapshot['validated'], False)
 
+        self.log.info("Check that loading the snapshot again will fail because there is already an active snapshot.")
```

Comment:
I also tested with putting this block before the above for loop, and the tests passes. I wanted to see what would happen if we load another snapshot right after loading one.